test/models/team.test.js

```diff
@@ -0,0 +1,165 @@
+const crypto = require('crypto')
+
+const utils = require('../utils.js')
+const { mongoose } = utils
+
+describe('Team', () => {
+  // models will be accessable after global 'before' hook runned (on util.js)
+  const { User, Team, Page, PageOwner } = utils.models
+
+  const conn = mongoose.connection
+  let createdUsers = []
+
+  const createTeam = (...users) => {
+    return Team.create({
+      handle: crypto.randomBytes(16).toString('hex'),
+      users,
+    })
+  }
+  const createPage = ({ path = `/random/${crypto.randomBytes(16)}`, user = null } = {}) => {
+    const p = new Page({
+      path,
+      grant: Page.GRANT_PUBLIC,
+      grantedUsers: user ? [user._id] : createdUsers.map(user => user._id),
+      creator: user || createdUsers[Math.floor(createdUsers.length * Math.random())],
+    })
+    return p.save()
+  }
+
+  beforeAll(async () => {
+    const userFixture = [
+      { name: 'Anon 3', username: 'anonymous3', email: 'anonymous3@example.com' },
+      { name: 'Anon 4', username: 'anonymous4', email: 'anonymous4@example.com' },
+    ]
+    createdUsers = await testDBUtil.generateFixture(conn, 'User', userFixture)
+  })
+
+  afterAll(async () => {
+    await User.remove({})
+  })
+
+  afterEach(async () => {
+    await Promise.all([
+      ...createdUsers.map(u => Team.findByUser(u).remove()),
+      Page.remove({
+        creator: {
+          $in: createdUsers.map(user => user._id),
+        },
+      }),
+    ])
+  })
+
+  /**
+   * class methods
+   */
+
+  describe('#findByUser', () => {
+    it('Find teams by user collectly', async () => {
+      await createTeam(...createdUsers)
+      const team0 = await createTeam(createdUsers[0])
+      const team1 = await createTeam(createdUsers[1])
+
+      const teamsRelatedTo0 = await Team.findByUser(createdUsers[0])
+      expect(teamsRelatedTo0).toHaveLength(2)
+      // ここらへんの assert うまいことできんかな
+      expect(teamsRelatedTo0.map(team => team._id.toString())).toEqual(expect.not.arrayContaining([team1._id.toString()]))
```

Comment:
If writing shorter, you can write followings.
```js
expect(teamsRelatedTo0.every(team => !team._id.equals(team1._id))).toBe(true)
```
However, I'd rather use `.arrayContaining` than `.toBe`.

Reply:
Ya, we can use custom expect with https://github.com/facebook/jest/issues/256 workaround, but ... not clever.... So I uses your shorter suggestion!